frontend/src/components/navigations/side-nav-bar/SideNavBar.jsx

```diff
@@ -147,6 +144,10 @@ const SideNavBar = ({ collapsed }) => {
     },
   ];
 
+  if (getMenuItem && appDeployment) {
+    data[0].subMenu.splice(1, 0, getMenuItem.default(orgName));
```

Comment:
why do we require orgaName as a parameter here.? @athul-rs

Reply:
Using this in the plugin  to call the backend url where `org_id` is required.